src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -33,14 +33,12 @@
 #include <mlpack/methods/ann/activation_functions/poisson1_function.hpp>
 #include <mlpack/methods/ann/activation_functions/gaussian_function.hpp>
 
-#include <boost/test/unit_test.hpp>
-#include "test_tools.hpp"
+#include "catch.hpp"
+#include "test_catch_tools.hpp"
```

Comment:
Looks like we can remove the header file, because we don't use `CheckMatrices`.

Reply:
Agreed

Reply:
Done in 2bb130e.